src/Sql/Sql/help/New-AzSqlInstance.md

```diff
@@ -317,6 +317,34 @@ ZoneRedundant            : true
 
 This command creates a new instance with external administrator properties and azure active directory only authentication enabled.
 
+### Example 9: Create a new instance with TDE CMK
+```powershell
+New-AzSqlInstance -Name managedInstance1 -ResourceGroupName ResourceGroup01 -Location westcentralus -AdministratorCredential (Get-Credential) -SubnetId "/subscriptions/xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx/resourceGroups/resourcegroup01/providers/Microsoft.Network/virtualNetworks/vnet_name/subnets/subnet_name" -LicenseType LicenseIncluded -StorageSizeInGB 1024 -VCore 16 -SkuName GP_Gen4 -DnsZonePartner "/subscriptions/xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx/resourceGroups/resourcegroup01/providers/Microsoft.Sql/managedInstances/partnerServerForDnsZone" -AssignIdentity -IdentityType "UserAssigned" -PrimaryUserAssignedIdentityId "Primary UserManaged identity id" -UserAssignedIdentityId "UserManaged identity id" -KeyId "AzureKeyVault Uri"
```

Comment:
Let's add dummy resource ids for UserAssignedIdentityId and PrimaryUserAssignedIdentityId 
Let's add dummy AKV URLs for key id